README.md

```diff
@@ -337,7 +337,7 @@ The worker count should be set to 1 if the input data requires a Bio-Formats rea
 This is not a common case, but is a known issue with Imaris HDF data in particular.
 
 In general, expect to need to tune the above settings and measure
-relative performance. See the [Compression options section](#compression-options) above for more information on `--compression` and `--compression-options`.
+relative performance. See the [Compression options section](#compression-properties) above for more information on `--compression` and `--compression-properties`.
```

Comment:
This link work work.
the heading in line 136 is still options.

Reply:
Since the section discusses both available compressions and compression properties, my vote would be to keep the title as it is and fix the anchor link back to `#compression-option`

Reply:
Done in 1143228.